interactive/interactive/tools.py

```diff
@@ -1,12 +1,13 @@
 import os
 import json
-from django.conf import settings
-from django.http import HttpResponse
+# from django.conf import settings
+# from django.http import HttpResponse
 from .unit_dict import *
 from .conversion_dict import *
 from .converter_class import Unit
 
 
+base_dir = '/music-box-interactive/interactive'
```

Comment:
why is this needed? this will only work if the server is running in a docker container built with the Docker file we provide (i.e., with the `music-box-interactive/` folder in the root directory). It seems to me it would be better to use the `settings.BASE_DIR` to allow others (or us) to install differently if needed.